modules/ossm-kiali-service-mesh.adoc

```diff
@@ -14,4 +13,4 @@ Installing Kiali  via the Service Mesh on {product-title} differs from community
 * Ingress has been enabled by default.
 * Updates have been made to the Kiali ConfigMap.
 * Updates have been made to the ClusterRole settings for Kiali.
-* Do not edit the ConfigMap or the Kiali custom resource files as those changes might be overwritten by the {ProductShortName} or Kiali Operators. All configuration for Kiali running on {ProductName} is done in the `ServiceMeshControlPlane` custom resource file and there are limited configuration options. Updating the Operator files should be restricted to those users with `cluster-admin` privileges. If you use {product-dedicated}, updating the operator files should be restricted to those users with `dedicated-admin` privileges.
+* Do not edit the ConfigMap, as your changes might be overwritten by the {ProductShortName} or Kiali Operator. Resources that are managed by the Kiali Operator have a `app.kubernetes.io/name` label with the value `kiali` or the value of the Kiali CR's `spec.deployment.instance_name`. Updating the Operator resources should be restricted to those users with `cluster-admin` privileges. If you use {product-dedicated}, updating the Operator resources should be restricted to those users with `dedicated-admin` privileges.
```

Comment:
Sorry @JStickler - I just looked at the code (see [here](https://github.com/kiali/kiali-operator/blob/f6f8adad50d167b722ca504b315a8ced030ffc30/roles/default/kiali-deploy/tasks/main.yml#L570)) and I was slightly wrong. I have to change my previous suggestion. The `app.kubernetes.io/name` field is always `kiali` (regardless of the instance name setting). So we should change this to:

```suggestion
* Do not edit the ConfigMap, as your changes might be overwritten by the {ProductShortName} or Kiali Operator. Resources that are managed by the Kiali Operator have a `app.kubernetes.io/name` label with the value `kiali`. Updating the Operator resources should be restricted to those users with `cluster-admin` privileges. If you use {product-dedicated}, updating the Operator resources should be restricted to those users with `dedicated-admin` privileges.
```

Reply:
Thanks for double-checking.  Updated.